lib/actions-oclif/push.ts

```diff
@@ -121,8 +121,10 @@ export default class PushCmd extends Command {
 			char: 's',
 		}),
 		emulated: flags.boolean({
-			description: 'Force an emulated build to occur on the remote builder',
-			char: 'f',
+			description: stripIndent`
+				Don't use native ARM servers; force QEMU ARM emulation on Intel x86-64
+				servers during the image build (balenaCloud).`,
+			char: 'e',
```

Comment:
Reverting `'e'` as it was until #1960 and as documented online, e.g.: https://www.balena.io/docs/learn/deploy/deployment/#--emulated--e

While at it, I thought of rewording the option description for additional clarity and information.